.env.dev

```diff
@@ -17,3 +17,5 @@ SESSION_SECRET_KEY=my-secret-key
 # Path where all logs for teuthology-suite or teuthology-kill would be collected
 ARCHIVE_DIR=/archive_dir/
 TEUTHOLOGY_PATH=/teuthology
+
+TEUTHOLOGY_API_SQLITE_URI=
```

Comment:
I think we can add here `sqlite:////teuthology_api/teuthology.db` as value . The above two paths are also for container setup. What do you think?

Reply:
I left it because I wasn't sure what the actual path for the database would be. If we're keeping `teuthology_api/teuthology.db` as the path, I'll modify the last commit and update the `.env.dev`